### PR TITLE
New: post mockup

### PR DESCRIPTION
**Pull request by pronguen**
Adjust publication date when deploying

### CODE REVIEW

File: content/post/190801-mockups.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating mockups for SONAR"
+date: 2018-08-01T17:41:58+02:00
```
Comment (iGormilhit) [inline content/post/190801-mockups.md]: The year is kind of surprising. :slightly_smiling_face:

File: content/post/190801-mockups.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+draft: false
+tags: ["mockups", "user interface", "user-centered design"]
+---
+
+The work on designing the user interface of SONAR is progressing. According to the adopted methodology (presented in the beginning of the [post about personas](/post/personas/)), we are proceeding in five layers: 1. Strategy, 2. Scope, 3. Structure, 4. Skeleton and 5. Surface.
```
Comment (iGormilhit) [inline content/post/190801-mockups.md]: Should the layers be an ordered list?

File: content/post/190801-mockups.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Then we organised these scenarios logically, creating a tree for information architecture and a basic navigation map (layer 3: *Structure*).
+
+For the layer 4 *Skeleton*, we started to create 3-4 mockups, each simulating one scenario.
+
+### Example of one SONAR mockup
+
+The mockup presented below addresses the workflow of uploading a full-text publication as a researcher (IR as a service feature).
+
+<img class="image fit" src="/images/upload_publication.gif" alt="Mockups of uploading a full-text publication" title="Mockups of uploading a full-text publication" >
```
Comment (iGormilhit) [inline content/post/190801-mockups.md:36]: FYI, there's a HUGO shortcode for pictures: https://gohugo.io/content-management/shortcodes/#figure
It helps a lot (I think).

**Review by iGormilhit — APPROVED** [review]

File: content/post/190801-mockups.md (hunk 2 above)
Comment (pronguen) [reply content/post/190801-mockups.md]: No, as it appears in the chapeau of the article.

File: content/post/190801-mockups.md (hunk 3 above)
Comment (pronguen) [reply content/post/190801-mockups.md:36]: As it is a GIF and we had layout problems last time, I just copied the code of the other page.

**Pull request merged**